### DISCUSSION TITLE
[savestate] gives plugin error

### DISCUSSION THREAD

**Discussion by porres** [pierreguillot/Camomile#279] [Q&A] [answered]
I have an abstraction with [savestate] and it includes lines beginning with **#A** in the file. Camomile complains and says '#A object not found'. The patch then doesn't behave as expected. We're using camomile 1.0.7 with support for ELSE externals, built and saved with Vanilla 0.51-4. Tested on macOS 10.14.6 and Live 11.

![IMAGE 2021-10-30 20:55:24](https://user-images.githubusercontent.com/5360142/139561503-8072a202-996a-478e-9207-3d39195b1f59.jpg)

**Comment by pierreguillot** [comment]
I tested with Camomile 1.0.7 and it works well. There could be a problem in your patch or you might have introduce a bug when compiling your own version of Camomile. I can't help you sorry.

**Comment by porres** [comment]
We didn't really change anything in our version, but I can try a full vanilla version with just regular camomile

**Comment by pierreguillot** [comment]
>  can try a full vanilla version with just regular camomile

Yes, that would be better. 

And providing your patch, the plugin text file, etc. would be helpful ;)

**Comment by porres** [comment]
the thing is that I see this was a issue that was reported before, and you say it got fixed in 1.0.7, I wonder if maybe this wasn't fully taken care of

**Reply by pierreguillot** [reply]
The issue was a crash, it was fixed and the fix was validated by the reporter so I really doubt this is related. The object seems to work fine so I think, this is related to your patch (perhaps, you forget to add the abstraction that uses savestate for instance?). But the best would be that you give further information (such as the patches you use, the plugin text file, etc.) so that people can try here to debug and to help you.

**Comment by porres** [comment, marked as answer]
> perhaps, you forget to add the abstraction that uses savestate for instance?

nope, i'm using an abstraction from else that uses savestate, but i'll further investigate
